frontend/static_vol/js/modules/modalContents.js

```diff
@@ -32,7 +32,7 @@ const receiveMatchRequest = (args) => `
 `;
 
 const waitForOpponent = (args) => `
-    <section class="blockModal" data-modal-request_id="${args.request_id}" data-modal-username="${args.username}">
+    <section class="blockModal" data-modal-game_name="${args.game_name}" data-modal-number_of_players="${args.number_of_players}" data-modal-match_type="${args.MatchType}">
```

Comment:
```suggestion
    <section class="blockModal" data-modal-game_name="${args.game_name}" data-modal-match_type="${args.MatchType}">
```
`number_of_players`、`args`に定義されてない、かつ使用箇所なさそうなので削除で良いかと